README.md

```diff
@@ -49,11 +49,11 @@ go get github.com/onflow/flow-go-sdk
 
 ### Generating Keys
 
-Flow uses [ECDSA key pairs](https://en.wikipedia.org/wiki/Elliptic_Curve_Digital_Signature_Algorithm) 
+Flow uses [ECDSA](https://en.wikipedia.org/wiki/Elliptic_Curve_Digital_Signature_Algorithm) 
 to control access to user accounts. Each key pair can be used in combination with
 the SHA2-256 or SHA3-256 hashing algorithms.
 
-Here's how to generate an ECDSA private key on the P-256 curve:
+Here's how to generate an ECDSA private key for the P-256 (secp256r1) curve:
```

Comment:
A private key is actually just a number and is not on any curve (unlike the public key).